calico-cloud/reference/resources/kubecontrollersconfig.mdx

```diff
@@ -86,12 +86,12 @@ The node controller automatically cleans up configuration for nodes that no long
 
 ### Template
 
-| Field                     | Description                                                                         | Accepted Values     | Schema                              | Default  |
-| ------------------------- | ----------------------------------------------------------------------------------- | ------------------- | ----------------------------------- | -------- |
-| generateName              | Unique name used as suffix for host endpoints created based on this template        | Alphanumeric string | string                              |          |
-| nodeSelector              | Selects the nodes for which this template should create host endpoints              |                     | [Selector](#selectors)              | all()    |
-| interfaceCIDRs            | List of networks specified in CIDR notation                                         | List of valid CIDRs | List string                         |          |
-| labels                    | Labels to be added to generated host endpoints matching this template               |                     | map of string key to string values  |          |
+| Field                     | Description                                                                                                                         | Accepted Values     | Schema                              | Default  |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------------- | ----------------------------------- | -------- |
+| generateName              | Unique name used as suffix for host endpoints created based on this template                                                        | Alphanumeric string | string                              |          |
+| nodeSelector              | Selects the nodes for which this template should create host endpoints                                                              |                     | [Selector](#selectors)              | all()    |
+| interfaceCIDRs            | Used to determine which addresses from node.Spec.Addresses, IPv4/IPv6 tunnel addresses and Wireguard addresses will be included in the generated HostEndpoint if they are within the specified list of CIDRs. If no interface matches, the HostEndpoint will not be created | List of valid CIDRs | List string                         |          |
```

Comment:
How about this? 

"This configuration defines which IP addresses from a node's specification (including standard, tunnel, and WireGuard IPs) are eligible for inclusion in the generated HostEndpoint. IP addresses must fall within the provided CIDR ranges to be considered. If no address on the node matches the specified CIDRs, the HostEndpoint creation is skipped."